### PR TITLE
Added nightly XF

### LINKED ISSUE #51: Unable to use with XF version 4.7
Message: 
    System.TypeLoadException : Method 'GetHash' in type 'Xamarin.Forms.Mocks.PlatformServices' from assembly 'Xamarin.Forms.Core.UnitTests, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null' does not have an implementation.
  Stack Trace: 
    MockForms.Init(String runtimePlatform, TargetIdiom idiom)

### PR DESCRIPTION
**Pull request by VladislavAntonyuk**
Fixes #51

### CODE REVIEW

File: Xamarin.Forms.Mocks/PlatformServices.cs [hunk 1]
```diff
@@ -50,6 +52,20 @@ public string GetMD5Hash(string input)
...
+            using (var hash = SHA256.Create())
+            {
+                var enc = Encoding.UTF8;
+                byte[] result = hash.ComputeHash(enc.GetBytes(input));
+                foreach (byte b in result)
+                    stringBuilder.Append(b.ToString("x2"));
+            }
+
+            return stringBuilder.ToString();
+        }
```
Comment (jonathanpeppers) [inline Xamarin.Forms.Mocks/PlatformServices.cs:67]: Ok, so it looks like `GetHash` is new in 4.7.x. A question, I don't know the answer to.

If we just add this method, but leave everything targeting Xamarin.Forms 4.6, will it fix 4.7.x?

You could  revert some of the files here, and CI will produce a new `.nupkg` you can test:

https://ci.appveyor.com/api/buildjobs/e3yvuauteo2bui9q/artifacts/Xamarin.Forms.Mocks.zip

Thanks!

Comment (VladislavAntonyuk) [reply Xamarin.Forms.Mocks/PlatformServices.cs:67]: Unfortunately it didn't help: https://ci.appveyor.com/project/jonathanpeppers/xamarin-forms-mocks/builds/32642522/artifacts

Comment (VladislavAntonyuk) [reply Xamarin.Forms.Mocks/PlatformServices.cs:67]: that method is not found in interface where so, we have to use XF 4.7

**Comment by jonathanpeppers** [comment]
@VladislavAntonyuk thanks, I'll merge this after there is a 4.7 preview. Then we can push a Xamarin.Forms.Mocks 4.7.0.1-pre.

I spoke with the Xamarin.Forms team and it should be "pretty soon", but the release date isn't public.

**Comment by VladislavAntonyuk** [comment]
Ok, I think next week on Tuesday it will be released. I still want to leave availability to use nightly build from the main repository (without creating forks), we can prepare NuGet package to release on the earlier step by implementing necessary changes ASAP and then just updating package version to stable

**Comment by VladislavAntonyuk** [comment]
@jonathanpeppers updated to 4.7.0.773-pre1

**Comment by jonathanpeppers** [comment]
This should be on nuget shortly: https://www.nuget.org/packages/Xamarin.Forms.Mocks/4.7.0.1-pre

**Pull request merged**